components/mediation-admin/org.wso2.carbon.proxyadmin/src/main/java/org/wso2/carbon/proxyadmin/service/ProxyServiceAdmin.java

```diff
@@ -87,7 +88,12 @@ public String enableStatistics(String proxyName) throws ProxyAdminException {
                 } else {
                     proxy.getAspectConfiguration().enableStatistics();
                 }
-                persistProxyService(proxy);
+                CAppArtifactDataService cAppArtifactDataService = ConfigHolder.getInstance().
+                        getcAppArtifactDataService();
+                if (!cAppArtifactDataService.isArtifactDeployedFromCApp(getTenantId(), ServiceBusConstants.PROXY_SERVICE_TYPE + File.separator + proxyName)) {
```

Comment:
ServiceBusConstants.PROXY_SERVICE_TYPE + File.separator + proxyName used in many places. Use variable.